src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -146,17 +146,45 @@ C_FUNC(\Name\()_End):
 
         __PWTB_StackAlloc = __PWTB_TransitionBlock
 
+       // Make the stack operation compatible with GDB and LIBUNWIND
+       push {r7, lr}
+       mov r7, sp
+       sub sp, #((__PWTB_StackAlloc) + 4 * 4 + 9 * 4)
+
         .ifnc \pushArgRegs, DoNotPushArgRegs
-                PUSH_ARGUMENT_REGISTERS
```

Comment:
Sure. after getting positive test results, I'll clean up the code as well.
Right now, I just wanted to share the idea.